crates/ra_assists/src/utils.rs

```diff
@@ -1,10 +1,84 @@
 //! Assorted functions shared by several assists.
 
 use ra_syntax::{
-    ast::{self, make},
-    T,
+    ast::{self, make, NameOwner},
+    AstNode, T,
 };
 
+use hir::db::HirDatabase;
+
+use rustc_hash::FxHashSet;
```

Comment:
We generally don't leave blank lines between uses of external crate. The exception being imports from `std`

Reply:
Resolved, the highlighted two `use`. I wasn't sure if the block above it should also exclude the newline, but you didn't highlight it so I assumed no. Feel free to have me adjust accordingly.